polyglot.js

```diff
@@ -340,6 +344,17 @@ class PolyGlot {
             div.remove();
             return dims;
         }
+         // allow OOC talking
+        game.settings.register("polyglot", "allowOOC", {
+            name: "Allow talking OOC",
+            hint: "Lets the GM to use different languages out of character",
```

Comment:
The sentence is not proper english "Lets the GM to use" should be "Lets the GM use" or "allows the GM to use". The "Allow talking OOC" is also not proper english.

I'd suggest : 
name: "Activate on OOC chat messages"
hint: "Allows the GM to use different languages when speaking out of character"

These also should be wrapped in calls to `game.i18n.localize` and the strings put in lang/en.json, but since this module doesn't have any translations in the first place, that's on me to port it to use them.